<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeepLabel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="imageDisplayLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="imageInfoLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="1" column="0">
       <widget class="QGroupBox" name="labelGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Label information</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="1">
          <widget class="QPushButton" name="removeLabelsForwardButton">
           <property name="text">
            <string>Remove next labels</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Class</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="removeImageLabelsButton">
           <property name="text">
            <string>Remove image labels</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="nextInstanceButton">
           <property name="text">
            <string>Next instance</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelID">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelClass">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="navigationGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Navigation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="4" column="1" colspan="2">
          <widget class="QPushButton" name="nextUnlabelledButton">
           <property name="text">
            <string>Next Unlabelled</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QPushButton" name="changeImageButton">
           <property name="text">
            <string>Jump to image</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="imageIndexLabel">
           <property name="text">
            <string>Image: 0/0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="propagateCheckBox">
           <property name="text">
            <string>Track labels</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QProgressBar" name="imageProgressBar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="format">
            <string>Image %v/%m </string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSpinBox" name="imageNumberSpinbox">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="imageGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Image Attributes</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QLabel" name="filetypeLabel">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Instances labelled:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="dimensionsLabel">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Filename:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Dimensions:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="imageBitDepthLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Filetype: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="filenameLabel">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="removeImageButton">
           <property name="text">
            <string>Remove image</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Bit Depth</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="sizeLabel">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="instanceCountLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Size on disk: </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionMerge_Project"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>Images</string>
    </property>
    <addaction name="actionAdd_image"/>
    <addaction name="actionAdd_image_folder"/>
    <addaction name="actionAdd_image_folders"/>
    <addaction name="actionAdd_video"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionWrap_images"/>
    <addaction name="separator"/>
    <addaction name="actionJump_forward"/>
    <addaction name="actionJump_backward"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuConfigure_Classes">
     <property name="title">
      <string>Configure Classes</string>
     </property>
     <addaction name="actionAdd_Remove_Class"/>
     <addaction name="actionAdd_Remove_Attributes"/>
    </widget>
    <addaction name="menuConfigure_Classes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImages"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreviousImage"/>
   <addaction name="actionNextImage"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionimport_labels"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPreviousImage">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="toolTip">
    <string>Previous Image</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionNextImage">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAdd_image">
   <property name="text">
    <string>Add image(s)</string>
   </property>
  </action>
  <action name="actionAdd_image_folder">
   <property name="text">
    <string>Add image folder</string>
   </property>
  </action>
  <action name="actionAdd_class">
   <property name="text">
    <string>Add class</string>
   </property>
  </action>
  <action name="actionDraw_Tool">
   <property name="text">
    <string>Draw</string>
   </property>
   <property name="toolTip">
    <string>Create rectangular labels</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="actionSelect_Tool">
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select labels for viewing/deletion</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionTwo_click">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Two-click</string>
   </property>
  </action>
  <action name="actionDrawg">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Drag</string>
   </property>
  </action>
  <action name="actionWrap_images">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wrap images</string>
   </property>
  </action>
  <action name="actionInit_Tracking">
   <property name="text">
    <string>Init Tracking</string>
   </property>
   <property name="toolTip">
    <string>Initialise object tracker with this image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionPropagate_Tracking">
   <property name="text">
    <string>Propagate Tracking</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export labels</string>
   </property>
  </action>
  <action name="actionRefine_boxes">
   <property name="text">
    <string>Refine boxes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAdd_video">
   <property name="text">
    <string>Add video(s)</string>
   </property>
  </action>
  <action name="actionAdd_image_folders">
   <property name="text">
    <string>Add image folders</string>
   </property>
  </action>
  <action name="actionJump_forward">
   <property name="text">
    <string>Jump forward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionJump_backward">
   <property name="text">
    <string>Jump backward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionNext_Image">
   <property name="text">
    <string>Next Image</string>
   </property>
  </action>
  <action name="actionPrevious_Image">
   <property name="text">
    <string>Previous Image</string>
   </property>
  </action>
  <action name="actionSetup_detector">
   <property name="text">
    <string>Setup detector</string>
   </property>
  </action>
  <action name="actionDetect_Objects">
   <property name="text">
    <string>Detect Objects</string>
   </property>
  </action>
  <action name="actionSet_threshold">
   <property name="text">
    <string>Set confidence threshold</string>
   </property>
  </action>
  <action name="actionDetect_project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run on project</string>
   </property>
  </action>
  <action name="actionSet_NMS_threshold">
   <property name="text">
    <string>Set NMS threshold</string>
   </property>
  </action>
  <action name="actionMerge_Project">
   <property name="text">
    <string>Merge Project</string>
   </property>
  </action>
  <action name="actionAdd_Remove_Class">
   <property name="text">
    <string>Add/Remove Class</string>
   </property>
  </action>
  <action name="actionAdd_Remove_Attributes">
   <property name="text">
    <string>Add/Remove Attributes</string>
   </property>
  </action>
  <action name="actionimport_labels">
   <property name="text">
    <string>import_labels</string>
   </property>
   <property name="toolTip">
    <string>import labels from a json file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
